adoc/chapters/programming_interface.adoc

```diff
@@ -14062,7 +14063,7 @@ of 10 elements of [code]#int# type, the host pointer must at least have
 A special case is the [code]#update_host# member function.
 This member function only requires an accessor, and instructs the runtime to update
 the internal copy of the data in the host, if any. This is particularly
-useful when users use manual synchronization with host pointers, e.g.
```

Comment:
why removing `host pointers`?

Reply:
I was trying to avoid the phrase "synchronization with host pointers".  I felt it was ambiguous whether it meant "manual synchronization [with host pointers]" or "manual [synchronization with host pointers]", and neither is good.  The former is confusing because host pointers don't provide synchronization, and the latter is potentially confusing because it's quite close to "synchronize with".

I convinced myself that the mention of "host pointers" didn't add anything to the sentence, so the simplest fix was to take it out.  If we're trying to clarify what "manual synchronization" is, I think the most important part to highlight is that we're talking about the buffer constructor which takes a mutex.  If somebody looks at the description of that constructor, they'll see how it relates to host pointers.

Reply:
Agree that the term `manual synchronization` is confusing, as we don't use the term `synchronization` anymore.
If we use particularly useful, I guess we can go deep into the `particularity` and just wrote:

>This is particularly useful when users use the mutex objects on the [code]#buffer# constructors.

or something around those line

Reply:
To make things even more confusing, I think we _could_ use "manual synchronization" here, because a mutex _is_ a synchronization primitive. :laughing:

But I'm not opposed to changing it to something like you suggested.  Maybe:

> This is particularly useful when used in conjunction with the [code]#buffer# constructor overloads which accept mutex objects.

What do you think?

Reply:
Perfect! Far clearer. I finally understood the sentence! :D

Reply:
New wording in 19fcedc59740213a52d6f5137103463a50c4baa3.